Backend/Hopsy/src/main/java/hopsy/BeerDBController.java

```diff
@@ -0,0 +1,37 @@
+package hopsy;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BeerDBController {
+    public BeerDBController(){}
+
+    @RequestMapping("/create-beerDB")
+    public static boolean createDB(@RequestBody String beers) {
+        MongoClient usrMC = DBUtils.getusrMC();
+        MongoDatabase db = usrMC.getDatabase("BeerDB");
+        MongoCollection<Document> dbCollection = db.getCollection("beers");
+
+        Document doc = new Document("beers", beers);
+        DBUtils.insertDoc(dbCollection, doc);
+
+        return true;
+    }
+
+    @RequestMapping("/get-beerDB")
+    public static String getBeerDB() {
+        MongoClient usrMC = DBUtils.getusrMC();
+        MongoDatabase db = usrMC.getDatabase("BeerDB");
```

Comment:
If you want you could make "beers" and "BeerDB" consts but they aren't used that much so its up to you 👍